tests/installation/prepare_maintenance_env.pm

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Prepare test repositories for children jobs.
+# - Disable Nvidia repos that may have been automatically added as part of autoinstall.
```

Comment:
i am not sure if this is accurate. Can you confirm?

Reply:
apparently this was done twice, so in the autoyast profile might suffice:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a8142f8da6745b5b09e2d77369f4ae436ac4ab1c/tests/qa_automation/patch_and_reboot.pm#L38
but take into account that this file was copy/pasted from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_qam/15_installation.xml.ep
My initial suggestion was to clone the compact profile and put on top the templating, but it was done reusing that file above, so in this case we can optimize things later if something is not needed.

Reply:
Good catch, I actually removed the line since it's being removed in the autoyast. I'll remove it.